_help/README.md

```diff
@@ -0,0 +1,96 @@
+# Maintaining Help Section
+
+The Help section of login.gov a slightly different from others. It is the core of the site mobile navigation. 
```

Comment:
For my own clarity, could you elaborate on this a bit?

>It is the core of the site mobile navigation. 

Is the content of this folder the basis for generating navigation lists? Is that true just for the "Help" section, or site-wide?

Reply:
It's been a while, but I confirmed with Nick that the Help section is NOT the core site navigation 😉